### DISCUSSION TITLE
Login Background Image Not Showing

### DISCUSSION THREAD

**Discussion by luanaraujo** [requarks/wiki#7091] [Help / Questions] [answered]
Hi everyone, I changed the setting in the security tab to show a background image on my login page, but it isnt working.

![image](https://github.com/requarks/wiki/assets/50503859/99d65a4a-0523-492c-9ed6-fb839dc981f1)

I made the installation in a docker container following the tutorial on the Wiki.js get started page.
But when I go to the login page, the background is not showing
![image](https://github.com/requarks/wiki/assets/50503859/85b1ccb7-6428-49eb-a435-a1ac1d803b3e)

**Comment by fnafox** [comment, marked as answer]
Guest group needs to have read permissions to your asset.

**Reply by luanaraujo** [reply]
Thank you!
